.github/workflows/soak.yml

```diff
@@ -215,7 +215,7 @@ jobs:
     name: Soak analysis (${{ matrix.target }})
     if: ${{ github.actor != 'dependabot[bot]' }}
     runs-on: [self-hosted, linux, x64, soak] # could be general if we move away from miller
-    needs: [soak-baseline, soak-comparison]
+    needs: [compute-test-plan, soak-baseline, soak-comparison]
```

Comment:
I'd actually thought about adding it here, but decided it wasn't necessary since `soak-baseline` and `soak-comparison` requires it, but, on second thought, I do like the explicitness given this job does use the output of it.

Reply:
It turns out if it's not present the matrix reference fails. TIL

I'm actually working on putting all the captures in a single sample which removes the need for this, so I'll back it out once that's ready.

Reply:
Aha, interesting 🤔